x-pack/plugins/fleet/cypress/e2e/a11y/home_page.cy.ts

```diff
@@ -80,7 +80,7 @@ describe('Home page', () => {
         cy.getBySel(AGENT_FLYOUT.QUICK_START_TAB_BUTTON, { timeout: 15000 }).should('be.visible');
         cy.getBySel(AGENT_FLYOUT.ADVANCED_TAB_BUTTON).click();
       });
-      it('Select policy for fleet', () => {
+      it.skip('Select policy for fleet', () => {
```

Comment:
What's the reason for skipping here? Happy to fix in a follow-up rather than blocking this PR.

Reply:
Sorry @kpollich, I just have missed that during the cleanup, basically there were same issues at the beginning with `cypress-axe` plugin, so I have skipped all scenarios using `a11y`, but it seems everything works fine now